index.html

```diff
@@ -10,6 +10,8 @@
 </head>
 
 <body>
+
+
```

Comment:
What the fuck? Do we really need this empty line?

Reply:
![image](https://user-images.githubusercontent.com/40394400/170889663-7c884b5c-ecaa-4c8c-99a2-d65a45d12ac3.png)